tests/Unit/PrestaShopBundle/Command/ConfigCommandTest.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Unit\PrestaShopBundle\Command;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Adapter\Configuration;
+use PrestaShop\PrestaShop\Adapter\Language\LanguageDataProvider;
+use PrestaShop\PrestaShop\Adapter\LegacyContextLoader;
+use PrestaShopBundle\Command\ConfigCommand;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class ConfigCommandTest extends TestCase
+{
+    public function testGet(): void
+    {
+        $commandTester = $this->getCommandTester();
+
+        $this->assertEquals(ConfigCommand::STATUS_OK, $commandTester->execute(['action' => 'get', 'key' => 'CONFIG_TEST']));
```

Comment:
You should test with all added condition in the get method.
Try to use `XDEBUG_MODE=coverage` and see if you tested all use cases :wink:

Reply:
```
root@05ee984cbfb2:/var/www/html# XDEBUG_MODE=coverage vendor/bin/phpunit tests/Unit/PrestaShopBundle/Command/ConfigCommandTest.php
PHPUnit 8.5.16 by Sebastian Bergmann and contributors.

.....                                                               5 / 5 (100%)

Time: 50 ms, Memory: 6.00 MB

OK (5 tests, 10 assertions)
```

Can you explain how coverage works with php? Never used it with phpunit...

Reply:
now I know https://github.com/PrestaShop/docs/pull/1236